packages/codegen/generate.server.utils.js

```diff
@@ -233,6 +233,20 @@ function generateServerUtils (gql) {
         })
     }
 
+    async function updateMany (context, data, extraAttrs = {}, options = {}) {
+        if (!context) throw new Error('no context')
+        if (!data) throw new Error('no data')
+        _checkOptions(options)
+
+        return await execGqlWithoutAccess(context, {
+            query: gql.UPDATE_OBJS_MUTATION,
+            variables: { data: data.map( data => ({ id: data.id, data: { ...data.data, ...extraAttrs } })) },
```

Comment:
`data.map(data => )` Seems like name duplication, how about to make it more clear to read?